docs/machine-learning/how-to-guides/normalizers-preprocess-data-ml-net.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Preprocess training data with normalizers to use in data processing - ML.NET
+description: Learn how to use normalizers to preprocess training data for use in machine learning model building, training, and scoring with ML.NET
+ms.date: 11/07/2018
+ms.topic: how-to
+ms.custom: mvc
+#Customer intent: As a developer, I want to use normalizers to preprocess training data so that I can optimize it in machine learning model building, training, and scoring with ML.NET.
+---
+# Preprocess training data with normalizers to use in data processing - ML.NET
+
+ML.NET exposes a number of [parametric and non-parametric algorithms](https://machinelearningmastery.com/parametric-and-nonparametric-machine-learning-algorithms/).
+
+It's **not** as important which normalizer you choose as it is to **use** a normalizer when training linear or other parametric models.
+
+Always include the normalizer directly in the ML.NET learning pipeline, so it:
+
+- is only trained on the training data, and not on your test data,
+- is correctly applied to all the new incoming data, without the need for extra pre-processing at prediction time.
+
+Here's a snippet of code that demonstrates normalization in learning pipelines. It assumes the Iris dataset:
+
+```csharp
```

Comment:
I'll stop mentioning it on the other files, but the comment holds that all the samples should move to the samples repo.